Abstract.md

```diff
@@ -1,3 +1,3 @@
-This document contains a normative standard for designing APIs in the Dutch Public Sector.  
-[The Governance of this standard](https://publicatie.centrumvoorstandaarden.nl/api/adr-beheer/) is described in a [separate repository](https://github.com/Logius-standaarden/ADR-Beheermodel) and published by Logius.  
-This document is part of the *Nederlandse API Strategie*, which consists of [a set of documents](https://www.geonovum.nl/themas/kennisplatform-apis#APIStrategie).  
\ No newline at end of file
+This document contains a normative standard for designing APIs in the Dutch Public Sector.
+[The Governance of this standard](https://gitdocumentatie.logius.nl/publicatie/api/beheermodel/) is described by the API-Standaarden beheermodel in a [separate repository](https://github.com/Logius-standaarden/API-Standaarden-Beheermodel/) and published by Logius.
```

Comment:
```suggestion
The Governance of this standard is described by the [API-Standaarden beheermodel](https://gitdocumentatie.logius.nl/publicatie/api/beheermodel/) published by Logius.
```
Wellicht duidelijker om te zeggen dat de governance beschreven staat in het beheermodel i.p.v. in een repository.